src/main/java/com/google/sps/servlets/CasesDataServlet.java

```diff
@@ -43,23 +46,24 @@
   @Override
   public void init() {
     Collection<Report> reports = new ArrayList<>();
-    // Gets file to scan
-    Scanner scanner = null;
-    try {
-      File statsFile = new File(STATS);
-      scanner = new Scanner(statsFile);
-    } catch (FileNotFoundException e) {
-      System.out.println(e);
-    }
+    Scanner scanner = connectToData();
 
-    // Parses data set for coordinates and active location
+    // Parse the data set
+    String line = scanner.nextLine();
     while (scanner.hasNextLine()) {
-      String line = scanner.nextLine();
+      // Skip the first line (header)
```

Comment:
since you pushed I'm sure that it works, but it's unclear to me going through your code why you skipped the first line of the scanner in line 52. My guess is that the first line of the file is a header or something, but that's just a guess.